big_tests/tests/sm_SUITE.erl

```diff
@@ -193,9 +199,19 @@ required_modules(Config, Scope, Name) ->
                    stopped -> stopped;
                    ExtraOpts -> maps:merge(common_sm_opts(Config), ExtraOpts)
                end,
+    PingConfig = case Name of
+                    ping_timeout ->
+                        #{send_pings => true,
+                          ping_interval => ?PING_INTERVAL,
+                          ping_req_timeout => ?PING_REQUEST_TIMEOUT,
+                          timeout_action => kill};
+                    _ ->
+                        #{}
+                 end,
     Backend = mongoose_helper:mnesia_or_rdbms_backend(),
     [{mod_stream_management, config_parser_helper:mod_config(mod_stream_management, SMConfig)},
-     {mod_offline, config_parser_helper:mod_config(mod_offline, #{backend => Backend})}].
+     {mod_offline, config_parser_helper:mod_config(mod_offline, #{backend => Backend})},
+     {mod_ping, config_parser_helper:mod_config(mod_ping, PingConfig)}].
```

Comment:
as far as I understand we don't need to add mod_ping module for any other groups/testcases than ping_timeout